### PR TITLE
Add sample code for 24 segments bargraph

### PR DESCRIPTION
**Pull request by chevdor**

### CODE REVIEW

**Comment by chevdor** [comment]
Hi there, is anything blocking the merging of this PR?

**Comment by chevdor** [comment]
Hi @chalkers do you need any change in this PR or could it be merged?

File: README.md [hunk 1]
```diff
@@ -306,6 +306,82 @@ function main() {
...
 
+## Example Code for 24 Segment Bargrah
+
+![](docs/24segments.gif)
+
+
+```
+import { connect8x8BicolorMatrix } from '@thingssdk/ht16k33/espruino';
+var intervalId =0;
+//Squint a little and you can see the faces!
```
Comment (chalkers) [inline README.md]: This comment isn't really for this bargraph

File: README.md [hunk 2]
```diff
@@ -306,6 +306,82 @@ function main() {
 }
 ```
 
+## Example Code for 24 Segment Bargrah
```
Comment (chalkers) [inline README.md]: sp. Bargraph

File: README.md [hunk 3]
```diff
@@ -306,6 +306,82 @@ function main() {
...
+
+![](docs/24segments.gif)
+
+
+```
+import { connect8x8BicolorMatrix } from '@thingssdk/ht16k33/espruino';
```
Comment (chalkers) [inline README.md]: connect8x8BicolorMatrix is for Bi Color Displays

A more appropriate API would be BicolorBargraph

File: README.md [hunk 4]
```diff
@@ -306,6 +306,82 @@ function main() {
...
+    [[0b11111111, 0b00000000], [0b00001111, 0b00000000], [0b00001111, 0b00000000]],
+    [[0b11111111, 0b00000000], [0b00111111, 0b00000000], [0b00001111, 0b00000000]],
+    [[0b11111111, 0b00000000], [0b11111111, 0b00000000], [0b00001111, 0b00000000]],
+    [[0b11111111, 0b00000000], [0b11111111, 0b00000000], [0b00111111, 0b00000000]],
+    [[0b11111111, 0b00000000], [0b11111111, 0b00000000], [0b11111111, 0b00000000]],
+    ];
+
+//Run Example Code
+function main() {
+    const matrix = connect8x8BicolorMatrix()
```
Comment (chalkers) [inline README.md]: mayby `const bargraph` and again 8x8BicolorMatrix isn't descriptive API...

File: README.md [hunk 5]
```diff
@@ -306,6 +306,82 @@ function main() {
...
+
+    var index = 0;
+
+    //Draws Neutral Face after 1 second
```
Comment (chalkers) [inline README.md]: Comment not descriptive of what this is doing.

File: README.md [hunk 6]
```diff
@@ -306,6 +306,82 @@ function main() {
...
+    intervalId = setInterval(() => {
+        matrix.render(patterns[index++]);
+        if (index >= patterns.length) index = 0;
+    }, 250);
+    console.log('interval = ' + intervalId)
```
Comment (chalkers) [inline README.md]: Why?

**Review by chalkers — CHANGES_REQUESTED** [review]
Hi @chevdor 

Attached are some comments regarding this pull request.

It needs a little work. 

Thanks
Andrew

**Comment by chevdor** [comment]
Hi @chalkers 
I have pushed a fix that should address what you mentioned in your feedback.

**Comment by chevdor** [comment]
This PR has been ready for a while on my end. Can it be reviewed and either merged or closed please ?

**Comment by chevdor** [comment]
Dormant review...

**Pull request closed without merging**